tasks/main.yml

```diff
@@ -1,5 +1,16 @@
 ---
 # tasks file for ansible_filebeat
+
+- name: get the rpm package facts
+  package_facts:
+    manager: "auto"
+
+- name: uninstall old filebeat package
+  yum:
+    name: filebeat
+    state: absent
+  when: "ansible_facts.packages.filebeat[0].version != version"
+
 - name: Install filebeat from elastic url
```

Comment:
should we also do the rest of the tasks .. with conditional?

Reply:
like when: "ansible_facts.packages.filebeat[0].version != version"

Reply:
yay . Thought i did it :)